content/docs/tutorials/converting/congratulations@es.md

```diff
@@ -0,0 +1,26 @@
+---
+$title: ¡Felicitaciones!
+$order: 4
+---
+
+Ha terminado el tutorial y convertido correctamente un documento HTML en un documento de AMP.
+
+En este tutorial, aprendió a:
+
+- Reconocer las diferencias entre AMP y las páginas HTML tradicionales
+- Dominar los fundamentos de un sitio de AMP
+- Convertir una página HTML tradicional a AMP HTML
+- Validar una página AMP
+- Vincular el contenido de su AMP
+- Añadir datos estructurados para los motores de búsqueda
+
+
+## Próximos pasos
+
+Para obtener más información sobre AMP, explore estos recursos:
+
+- [AMP Start](https://www.ampstart.com/): Brinde estilo a sus páginas con plantillas pre-diseñadas y componentes
+- [AMP By Example](https://ampbyexample.com/): Aprende con ejemplos y demos en vivo
+- [AMP Channel on YouTube](https://www.youtube.com/channel/UCXPBsjgKKG2HqsKBhWA4uQw): Aprenda viendo videos específicos sobre AMP
+- [AMP HTML spec](/docs/reference/spec.html): Más información sobre el marcado HTML necesario en una página AMP
+- [AMP components](https://www.ampproject.org/docs/reference/components): Obtenga detalles sobre todos los componentes de AMP
```

Comment:
Change the link to  `/es/docs/reference/components.html`

Reply:
Done